config/src/main/java/com/glassdoor/planout4j/config/Planout4jConfigClasspathBackend.java

```diff
@@ -0,0 +1,101 @@
+package com.glassdoor.planout4j.config;
+
+import com.google.common.io.Files;
+import com.typesafe.config.Config;
+
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * A {@link Planout4jConfigBackend} which loads namespace content found on the classpath.
+ *
+ * @author rschatz
+ */
+public class Planout4jConfigClasspathBackend implements Planout4jConfigBackend {
+
+    /**
+     * Returns the name of the directory on the classpath which contains the namespace files.
+     * @return String
+     */
+    protected String getPlanoutSubdirectory() {
+        return "planout";
+    }
+
+    /**
+     * {@link Planout4jConfigClasspathBackend} does not need to be configured.
+     *
+     * @param config any {@link Config}
+     */
+    @Override
+    public void configure(Config config) { }
```

Comment:
The class path subdir can be a configurable parameter, but I guess this can wait for future refactoring if someone needs it